sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -781,3 +782,68 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function that returns the number of elements in outer Json Array.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(jsonArray) - Returns the number of elements in outer Json Array.",
+  arguments = """
+    Arguments:
+      * jsonArray - A JSON array is required as argument. `Analysis Exception` is thrown if any
+        other valid JSON expression is passed. `NULL` is returned in case of invalid JSON.
```

Comment:
Can you add two more spaces? See also the documentation at `ExpressionDescription`